Poker.BE/Poker.BE.DomainTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Poker.BE.DomainTests")]
```

Comment:
you should use `Poker.BE.Domain.Tests` csproject only. with the **dot** between `Domain` and `Test`
![image](https://cloud.githubusercontent.com/assets/9889268/26259971/28a3f774-3cd4-11e7-9e92-f130217683af.png)